src/main/java/ca/uhn/fhir/jpa/starter/service/HelperService.java

```diff
@@ -497,19 +495,18 @@ public ResponseEntity<LinkedHashMap<String, Object>> createGroups(MultipartFile
 			}
 		}
 
-		map.put("count", iteration);
-		if (invalidClinics.size() > 0) {
-			map.put("issues", invalidClinics);
+		if (map.size() > 0) {
+			map.put("uploadTaskStatus", "Failure");
```

Comment:
i see uploadTaskStatus here and somewhere down UploadTaskStatus. Keep it string constant.